pangea-sdk/pangea/pangea.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	version   = "v1.0.0"
+	version   = "v1.1.0"
 	userAgent = "Pangea Go " + version
```

Comment:
completely unrelated to this MR, do we want to be a little more user-agent like with a string like `pangea-go/<version>` as [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax) does?

I know requests uses `requests/2.27.0` or whatever.

Totally acknowledge that this could be [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality) though

Reply:
No, you are right. I'll change it

Reply:
okay, resolving this thread